### ISSUE TITLE
Overall Design Love

### ISSUE THREAD

**Issue by chriswhong** [chriswhong/ReallySimpleOpenData#3]
Need some help with UX, color scheme, and workflow for search/filtering.  Open to all suggestions!

**Comment by mjrouser** [comment]
Still looking for design help?  I just got things up and running.

**Comment by chriswhong** [comment]
Yes, definitely.  The overall workflow is basically the same as CKAN, just shoved into bootstrap and made with express + mongodb.  Help with bootstrap layouts, a logo, and some User Experience things would be grand.
